app/experimenter/experiments/tasks.py

```diff
@@ -129,6 +131,7 @@ def update_experiment_info():
             logger.info("Updating Experiment: {}".format(experiment))
             if experiment.normandy_id:
                 update_status(experiment)
+                send_experiment_ending_emails(experiment)
```

Comment:
@tiftran Yea, in this implementation I am relying on the calculated/proposed end date. Because we want to send the email 5 days ahead, we don't have another option, the way it seems to me?

Reply:
I think my question boils down to do we want to send ending emails for experiments with accepted status as well if they're not "live"?

Reply:
Oh I see so... if the experiment has been accepted for a very long time... and never goes live... but the proposed end date comes along.. 5 days out, and the email gets sent?

Reply:
🙇‍♀ apologies for the poorly worded question. Yes, ^ this was what I was kinda trying to say

Reply:
So apparently I don't really know the difference between accepted or live... what's the difference? Is is common for there to be a really large difference in time?

Reply:
I don't exactly know either, which is why this was a question 😅.
Maybe @jaredkerim can clarify...

Reply:
I believe "Accepted" is "Accepted by Normandy" in the UI, which is what we use to say that the recipe is in Normandy and ready to go, but not actually being sent to users. In most cases this is just a few minutes, though it can realistically be up to a week when we do recipe pre-writing.

An experiment that is "Accepted" but not "Live" for a long time seems weird to me, though technically possible. I would say that would be a workflow/process problem. Experiments should either move forward to Live or backwards to the previous state fairly quickly.

Either way, I think that these emails should only be sent for the live state, and not the accepted state.

Reply:
ok yeah, true, these just shouldn't be sent unless it's live any way ya slice it!

Reply:
I added a check for only calling send_emails if the experiment is live... can you check that this is the best placement for it? obviously you could throw in a if experiment.status == Experiment.STATUS_LIVE in several different places... just wondering if you think it'd look better or feel better or BE better somewhere else...